code/game/objects/items/weapons/tools/hammer.dm

```diff
@@ -74,11 +76,10 @@
 	icon = 'icons/obj/weapons.dmi'
 	icon_state = "mace"
 	item_state = "mace"
-
 	armor_penetration = ARMOR_PEN_DEEP
 	force = WEAPON_FORCE_DANGEROUS
-
 	tool_qualities = list(QUALITY_HAMMERING = 20)
+	spawn_tags = SPAWN_TAG_WEAPON
```

Comment:
everything in order but if I remember correctly there is no weapons spawner, only one of knives. could you check?

Reply:
Checked, there is no weapon tag spawners, but the mace continues to spawn in-game through the "obj" and "item" tags regardless in random spawners.